jwst/ami/ami_analyze_step.py

```diff
@@ -157,6 +157,11 @@ def process(self, input):
         # pull out parameters that are strings and change to floats
         psf_offset = [float(a) for a in self.psf_offset.split()]
         rotsearch_parameters = [float(a) for a in self.rotation_search.split()]
+        # handle command-line None input interpreted as string
+        if affine2d in ["None", "none"]:
+            affine2d = None
+        if bandpass in ["None", "none"]:
+            bandpass = None
```

Comment:
This might be a little simpler/more robust if written like this:

```suggestion
        if str(affine2d).lower() == 'none':
            affine2d = None
        if str(bandpass).lower() == 'none':
            bandpass = None
```